tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -111,6 +114,19 @@ def is_favorites_list_displayed(self):
         return self.marionette.find_element(*self._favorites_list_locator).is_displayed()
 
     @property
+    def is_ice_list_displayed(self):
```

Comment:
`ice_list` is an ambiguous name. We don't really verify if there's a list, but we check if there's the icon that brings us to the actual list.

I'd rename this function to `is_ice_list_icon_displayed()`, like the locator.